### DISCUSSION TITLE
Slim CI/CD in Bitbucket Pipelines

### DISCUSSION THREAD

**Discussion by shippy** [dbt-labs/docs.getdbt.com#1215] [Archive]
I have a working prototype of slim CI/CD in Bitbucket working; it seems like there's general interest in the topic, so I'm writing it up. [Here's the outline.](https://docs.google.com/document/d/1SRZmUvYOTocrL04zZcQ5R0XZU0MatXM7LcUnMaf8Y94/edit?usp=sharing)

**Comment by shippy** [comment]
The outline now contains the full article.

**Comment by gwenwindflower** [comment]
Promoted to issue #1322
